prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,27 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.label import Label
+
+
+class DynamicLabelsApp(App):
+    """App for displaying dynamic labels for a list of names."""
+
+    def __init__(self, **name):
+        super().__init__(**name)
```

Comment:
I believe we are meant to use **kwargs for this.
Since this passes in all the arguments we don't need to specify the name of the argument.